src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -167,14 +166,13 @@ class instance_naxxramas : public InstanceMapScript
         ObjectGuid _lichkingGUID;
 
         // Controls
-        uint8 _horsemanKilled;
         uint32 _speakTimer;
         uint32 _horsemanTimer;
         uint32 _screamTimer;
         bool _hadThaddiusGreet;
         EventMap events;
         uint8 _currentWingTaunt;
-        bool _horsemanLoadDoneState;
+        uint8_t _horsemanLoadDoneState;
```

Comment:
uint8; This keeps a count of the loaded horsemen can you update the name to reflect this? e.g., `horsemenLoaded`